app/views/leads/index.html.erb

```diff
@@ -25,7 +25,7 @@
               </tr>
             </thead>
             <tbody>
-              <tr v-for="lead in filteredLeads" v-on:click="showEvents(lead)" v-bind:id="'lead-' + lead.id">
+              <tr v-for="lead in filteredLeads" v-on:click="showEvents(lead)" v-bind:id="'lead-' + lead.id" v-bind:style="lead.outreaches.length === 0 ? 'background-color:#f7c204;' : ''">
```

Comment:
Rather than inline style, we can use a `class`, but we'd have to add an `!important`. What do you think?